match/lib/match/storage/s3_storage.rb

```diff
@@ -45,16 +41,14 @@ def self.configure(params)
       end
 
       def initialize(s3_region: nil,
-                     s3_access_key: nil,
-                     s3_secret_access_key: nil,
                      s3_bucket: nil,
                      readonly: nil,
                      username: nil,
                      team_id: nil,
                      team_name: nil)
         @s3_bucket = s3_bucket
         @s3_region = s3_region
-        @s3_client = Fastlane::Helper::S3ClientHelper.new(access_key: s3_access_key, secret_access_key: s3_secret_access_key, region: s3_region)
```

Comment:
What is the reason for removing the access_key and secret_key from here? Isn't that needed for authenticating? I _think_ that access key and secret is needed for match to authenticate the user Into S3?

Reply:
I think I just missed revisiting this after adding back in the access_key/secret. Will update.